### PR TITLE
Fix single item cannot be activated in ItemList with SELECT_MULTI mode

### PR DESCRIPTION
**Pull request by Geequlim**
Now #4556 can be closed.

### CODE REVIEW

**Comment by akien-mga** [comment]
Ping @reduz, as you just changed this code so I guess the bug is a regression.

**Comment by Geequlim** [comment]
The bug #4556 was caused by #f909634832a72 in line  408

**Comment by Geequlim** [comment]
And I changed code [L461](https://github.com/godotengine/godot/blob/master/scene/gui/item_list.cpp#L461)

That may another bug. But I'm not sure.

**Comment by akien-mga** [comment]
@Geequlim Is this fix still needed after #4638? If so, could you rebase to fix the merge conflicts?

**Comment by Geequlim** [comment]
@akien-mga Yes it works.
I have merge conflicts since I think that way may still have something wrong with multi-select mode.

The code I have tested last week.

File: scene/gui/item_list.cpp [hunk 1]
```diff
@@ -458,16 +454,13 @@ void ItemList::_input_event(const InputEvent& p_event) {
 					bool selected = !items[j].selected;
 					select(j,false);
 					if (selected)
-						emit_signal("multi_selected",i,true);
+						emit_signal("multi_selected", j, true);
 				}
 			} else {
-
-				if (!mb.doubleclick && !mb.mod.command && select_mode==SELECT_MULTI && items[i].selectable && items[i].selected) {
+				if (!mb.mod.command && select_mode==SELECT_MULTI && items[i].selectable && items[i].selected) {
```
Comment (akien-mga) [inline scene/gui/item_list.cpp]: You dropped @volzhs's ``!mb.doubleclick``, is that on purpose or a mistake during the conflict resolution?

Comment (Geequlim) [inline scene/gui/item_list.cpp]: Yes I did that.

Comment (volzhs) [inline scene/gui/item_list.cpp]: It's actually fixed with modifying only 1 line in my PR for the same issue.

**Comment by Geequlim** [comment]
@akien-mga I have changed the code, now it just correct the index of `multi_selected` signal.

**Pull request closed without merging**